Firestore/Example/Tests/Remote/FSTSerializerBetaTests.mm

```diff
@@ -337,12 +336,8 @@ - (void)testEncodesSetMutation {
 }
 
 - (void)testEncodesPatchMutation {
-  FSTPatchMutation *mutation =
-      FSTTestPatchMutation(@"docs/1",
-                           @{ @"a" : @"b",
-                              @"num" : @1,
-                              @"some.de\\\\ep.th\\ing'" : @2 },
-                           nil);
+  FSTPatchMutation *mutation = FSTTestPatchMutation(
+      "docs/1", {{"a", "b"}, {"num", "1"}, {"some.de\\\\ep.th\\ing'", "2"}}, {});
```

Comment:
Same problem here: this must be {"num", 1} or something else that preserves the numeric nature. You'll need to audit all callers of FSTTestPatchMutation in this PR to revert this back.